modules/formulize/admin/formindex.php

```diff
@@ -643,6 +728,49 @@ function patch40() {
     }
 }
 
+function saveMenuEntryAndPermissionsSQL($formid,$appid,$i){
```

Comment:
This block also already exists in formindex.php and doesn't need to be added again.

Reply:
removed